Rails: Resource routing && Non existent routes (#81)
rootstrap/tech-guides: ruby/rails.md

  Don't use `match` to define any routes unless there is need to map multiple request types among `[:get, :post, :patch, :put, :delete]` to a single action using `:via` option.
<sup>[[link](#no-match-routes)]</sup>

* <a name="non-existing-routes"></a>

>   Don't create routes that are not being used. Use `:only` and `:except` to generate only the routes you actually need.

Reviewer: ```suggestion
  Don't create routes that won't be implemented. Use `:only` and `:except` to generate only the routes you actually need.
```
Not a blocker.

Author: Hmm ... I prefer the way it is.
"won't be imlemented" change the meaning. It means that I can define a route, because I `will` implement it later. And "later" sometimes translates into "never". And we don't want that, right?
